sql/telemetry_derived/error_aggregates/query.sql

```diff
@@ -0,0 +1,234 @@
+CREATE TEMP FUNCTION
+  udf_json_extract_int_map (input STRING) AS (ARRAY(
+    SELECT
+      STRUCT(CAST(SPLIT(entry, ':')[OFFSET(0)] AS INT64) AS key,
+             CAST(SPLIT(entry, ':')[OFFSET(1)] AS INT64) AS value)
+    FROM
+      UNNEST(SPLIT(REPLACE(TRIM(input, '{}'), '"', ''), ',')) AS entry
+    WHERE
+      LENGTH(entry) > 0 ));
+CREATE TEMP FUNCTION
+  udf_json_extract_histogram (input STRING) AS (STRUCT(
+    CAST(JSON_EXTRACT_SCALAR(input, '$.bucket_count') AS INT64) AS bucket_count,
+    CAST(JSON_EXTRACT_SCALAR(input, '$.histogram_type') AS INT64) AS histogram_type,
+    CAST(JSON_EXTRACT_SCALAR(input, '$.sum') AS INT64) AS `sum`,
+    ARRAY(
+      SELECT
+        CAST(bound AS INT64)
+      FROM
+        UNNEST(SPLIT(TRIM(JSON_EXTRACT(input, '$.range'), '[]'), ',')) AS bound) AS `range`,
+    udf_json_extract_int_map(JSON_EXTRACT(input, '$.values')) AS `values` ));
+CREATE TEMP FUNCTION udf_histogram_get_sum(histogram_list ANY TYPE, target_key STRING) AS (
+  (
+    SELECT
+      udf_json_extract_histogram(value).sum
+    FROM
+      UNNEST(histogram_list)
+    WHERE
+      key = target_key
+    LIMIT
+      1
+  )
+);
+CREATE TEMP FUNCTION udf_round_timestamp_to_minute(timestamp_expression TIMESTAMP, minute INT64) AS (
+  TIMESTAMP_SECONDS(
+    CAST((FLOOR(UNIX_SECONDS(timestamp_expression) / (minute * 60)) * minute * 60) AS INT64)
+  )
+);
+--
+-- Get pings from the last day from live tables, stable tables for older
+WITH crash_pings AS (
+  SELECT
+    *
+  FROM
+    telemetry_live.crash_v4
+  WHERE
+    DATE_DIFF(CURRENT_DATE, DATE(submission_timestamp), DAY) <= 1
+  UNION ALL
+  SELECT
+    *
+  FROM
+    telemetry_stable.crash_v4
+  WHERE
+    DATE_DIFF(CURRENT_DATE, DATE(submission_timestamp), DAY) > 1
+),
+main_pings AS (
+  SELECT
+    *
+  FROM
+    telemetry_live.main_v4
+  WHERE
+    DATE_DIFF(CURRENT_DATE, DATE(submission_timestamp), DAY) <= 1
+  UNION ALL
+  SELECT
+    *
+  FROM
+    telemetry_stable.main_v4
+  WHERE
+    DATE_DIFF(CURRENT_DATE, DATE(submission_timestamp), DAY) > 1
+),
+core_pings AS (
+  SELECT
+    *
+  FROM
+    telemetry.core_live
+  WHERE
+    DATE_DIFF(CURRENT_DATE, DATE(submission_timestamp), DAY) <= 1
+  UNION ALL
+  SELECT
+    *
+  FROM
+    telemetry.core
+  WHERE
+    DATE_DIFF(CURRENT_DATE, DATE(submission_timestamp), DAY) > 1
```

Comment:
I think these are more readable, easier to reason about, and more consistent with other queries in this repo when formatted with the partition field on the left and the (effectively) constant value on the right:
```suggestion
    DATE(submission_timestamp) < DATE_SUB(CURRENT_DATE, INTERVAL 1 DAY)
```

Reply:
Sure. I find both to be equally hard to reason about